core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -781,7 +789,7 @@ public static String getAcknowledgeModeAsString(int ackMode) {
 	/**
 	 * String-version of {@link #setAckMode(int)}
 	 */
-	@IbisDoc({"", "auto_acknowledge"})
+	@IbisDoc({"3", "Acknowledge mode, can be one of ('auto' or 'auto_acknowledge'), ('dups' or 'dups_ok_acknowledge') or ('client' or 'client_acknowledge')", "auto_acknowledge",})
 	public void setAcknowledgeMode(String acknowledgeMode) {
 
 		if (acknowledgeMode.equalsIgnoreCase("auto") || acknowledgeMode.equalsIgnoreCase("AUTO_ACKNOWLEDGE")) {
```

Comment:
En dit kan natuurlijk ook een enum worden in plaats van de if else if else if else if else if else